### PR TITLE
Feature/unit test frontend [Oleksii]

### PR DESCRIPTION
**Pull request by vglfr**

### CODE REVIEW

File: client/src/components/Cropper/Cropper.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default class Cropper extends React.Component<any> {
 
     public onBeforeFileLoad = (elem: any): void => {
         if (elem.target.files[0].size > this.FILE_SIZE) {
-            alert('File is too big!');
+            // alert('File is too big!');
```
Comment (anna-kuzii) [inline client/src/components/Cropper/Cropper.tsx]: comment?

File: client/src/components/Login/LoginContainer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        const match = { params: { token: 'sometoken' } };
+
+        // tslint:disable:trailing-comma
+        const store = configureStore()();
+        renderer = shallow(
+            <Provider store={store}>
+                <LoginContainer match={match} handleSubmit={handleSubmit} />
+            </Provider>
+        );
+        // tslint:enable:trailing-comma
```
Comment (anna-kuzii) [inline client/src/components/Login/LoginContainer.test.tsx]: too much this rules. Can we remove trailing-comma or change linter?

File: client/src/components/Cropper/Cropper.tsx (hunk 1 above)
Comment (vglfr) [reply client/src/components/Cropper/Cropper.tsx]: Deleted it.

File: client/src/components/Login/LoginContainer.test.tsx (hunk 2 above)
Comment (vglfr) [reply client/src/components/Login/LoginContainer.test.tsx]: Refactored.
I actually like trailing commas in multiline function arguments; this issue assessment sounds like a bug (probably tsx-related). Unfortunately tslint is on track to be merged with eslint and I'm not sure a) whether eslint works well with TypeScript b) whether tslint community would accept bug report, take apart act on it swiftly. Thus quick (though not very pretty) workaround seems to be an optimal solution.

Comment (anna-kuzii) [reply client/src/components/Login/LoginContainer.test.tsx]: I like trailing commas too. Can we change config?

Comment (vglfr) [reply client/src/components/Login/LoginContainer.test.tsx]: I can't see a simple solution to this proble. Tslint (falsely) identifies
```
return (
    <ReactElement />
);
```
as multiline function call and asks for a trailing comma after `/>`. Options are:
- comply (which breaks every style guide I could think of)
- disable block with `// tslint:disable...` (original solution)
- trick tslint with parenthesis coming right after `/>` (current solution)
- disable trailing commas for multiline function calls (approx. 20-30 changes in codebase)
- go for an (unlikely) bugfix in tslint
- attempt to switch to eslint (with unpredictable result)

**Pull request merged**